src/main/fc/settings.yaml

```diff
@@ -722,6 +722,12 @@ groups:
         default_value: "14"
         min: 4
         max: 255
+      - name: flip_over_after_crash_power_factor
```

Comment:
This should be guarded by `condition: USE_DSHOT` to make sure this setting is invisible on boards not supporting DSHOT.